docs/en/transform/split.mdx

```diff
@@ -35,6 +35,7 @@ This transform both supported by engine Spark and Flink.
 ### separator [string]
 
 Separator, the input string is separated according to the separator. The default separator is a space `(" ")` .
+Note: If you use some special characters in the separator, you need to escape it. e.g. "\\|"
```

Comment:
FYI, we using [docusaurus](https://docusaurus.io/) to reduce our docs, and it supports admonitions for a better show https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Thanks for your reminder.